README.md

```diff
@@ -14,6 +14,10 @@ LabGraph is a streaming framework built by the Facebook Reality Labs Research te
 ```
 pip install labgraph
 ```
+For Mac (supported version as of 02/02/2022):
+```
+pip install labgraph==2.0.0
```

Comment:
Let's keep "pip install labgraph"

Can you add a note in README.md
For more documentation on installation trouble shooting, please use this [link](https://github.com/facebookresearch/labgraph/tree/main/docs/installation).